cli/src/client/mod.rs

```diff
@@ -0,0 +1,44 @@
+use graphql_client::{GraphQLQuery, Response};
+use http::Uri;
+use reqwest::blocking;
+use std::error::Error;
+
+mod queries;
+
+static PROD_GQL_API_URL: &str = "https://engine-graphql.apollographql.com/api/graphql";
+
+pub struct Client {
+    personal_api_key: String,
```

Comment:
any reason not to call this `api_key`?

Reply:
I was thinking that somewhere else in the code we're going to ask the user to use their personal api key obtained from https://engine.apollographql.com/user-settings but actually this code specifically can take any type of API key so I'll rename.